### DISCUSSION TITLE
Withdraw

### DISCUSSION THREAD

**Discussion by Abuy1s2b3** [blockscout/blockscout#9276] [Support] [answered]
Withdraw from rarichain to artbitruim Chain hasn't arrived

Transaction ID:
0x2b5a78f02d7833f108dc85907f286ffc1794d8cf88c7dc1149d039fb5a5c47cb

**Comment by mesaifulop** [comment]
Withdraw from rarichain to arbitruim chain not arrived 😡🤬
Transection I'd:  0x0d863b9b50c3f14b09920871e5b68fa56394a3f806514612fcb975908109d489

**Comment by GimluCom** [comment, marked as answer]
This is the Blockscout Github. Please contact the chain directly for chain/network related issues.
